lib/SyTest/Homeserver.pm

```diff
@@ -341,6 +341,15 @@ sub _clear_db_pg
          $dbh->do( "DROP TABLE $tablename CASCADE" ) or
             die $dbh->errstr;
       }
+
+      foreach my $row ( @{ $dbh->selectall_arrayref(
+         "SELECT c.relname FROM pg_class c LEFT JOIN pg_namespace n ON n.oid = c.relnamespace WHERE relkind='S' AND n.nspname='public'"
+      ) } ) {
```

Comment:
If we're assuming we're in the public namespace then i wonder if we could just get away with doing a `DROP SCHEMA public CASCADE; CREATE SCHEMA public` or whatever.

But if this works I'm happy for us to do it like this for now.

Reply:
possibly, although I read that you could run into problems with which user ends up owning the schema if you do it that way.